src/Microsoft.AspNetCore.Mvc.Core/ModelBinding/Metadata/DefaultModelMetadataProvider.cs

```diff
@@ -84,6 +93,13 @@ private ModelMetadataCacheEntry CreateCacheEntry(ModelMetadataIdentity key)
             return new ModelMetadataCacheEntry(metadata, details);
         }
 
+        private ModelMetadata GetMetadataForObjectType()
+        {
+            var key = ModelMetadataIdentity.ForType(typeof(object));
+            var entry = CreateCacheEntry(key);
```

Comment:
I don't understand this feedback - how would returning the cached type-metadata for object be correct when we need to return a property-metadata?